ceph.spec.in

```diff
@@ -219,6 +219,7 @@ BuildRequires:	python3-Cython
 %if 0%{with make_check}
 %if 0%{?fedora} || 0%{?rhel}
 BuildRequires:	python%{_python_buildid}-cherrypy
+BuildRequires:	python%{_python_buildid}-routes
```

Comment:
Do we need to modify the debian control file?

Reply:
No need because when installing the python-cherrypy package in ubuntu, it also installs the python-routes package.